### DISCUSSION TITLE
"Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."

### DISCUSSION THREAD

**Discussion by inannan423** [community/community#44054] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

Today, this problem arises **again**!
![image](https://user-images.githubusercontent.com/83146544/211970709-c902de78-a23e-4de2-90d6-66b1b44d588e.png)
I have a Github Education license.On 1.9, many people's GitHub Copilot was not working until we received the renewal email again.  
But today this question bothers me again.  
![image](https://user-images.githubusercontent.com/83146544/211970866-dccc3714-fa12-4ca9-9fa5-60ee8b1d547f.png)

**Comment by IvanR-Vecerina** [comment]
Same Problem here, again after 3 days xD

**Comment by Akash1134** [comment, marked as answer]
Hi @inannan423 and @IvanR-Vecerina ,

Thank you for taking the time to report and we apologize for the temporary technical issue on our end that may have caused authentication failures. If you are a verified student please retry logging in to your IDE plugin, and Copilot should work as expected.

If you run into any further issues, please reply back in detail with any useful screenshots.

**Comment by ehddnr09** [comment]
Copilot extension doesn't work in vs code, so I went back to GitHub to log in and it says to subscribe. However, since I have a student account, I know that you can use it for free without having to subscribe, so please check this part again.

**Comment by AndersenMattias** [comment]
<img width="781" alt="Screenshot 2023-01-20 at 20 54 08" src="https://user-images.githubusercontent.com/64038954/213794066-cbf75344-a9af-481d-a2a2-cc28bef07988.png">


" Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."  I can't seem to purchase a subscription. Help..

**Comment by yogeshvar** [comment]
+1

**Comment by Razikus** [comment]
Hello i cant purchase subscription for copilot
i have only this " Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended"
